docs/webdriver-bidi.md

```diff
@@ -28,24 +24,79 @@ Below is an example of launching Firefox or Chrome with WebDriver BiDi:
 ```ts
 import puppeteer from 'puppeteer';
 
-const browser = await puppeteer.launch({
-  product: 'firefox', // or 'chrome'
-  protocol: 'webDriverBiDi',
+const firefoxBrowser = await puppeteer.launch({
+  product: 'firefox', // WebDriver BiDi is used by default.
+});
+const page = await firefoxBrowser.newPage();
+...
+await firefoxBrowser.close();
+
+const chromeBrowser = await puppeteer.launch({
+  product: 'chrome',
+  protocol: 'webDriverBiDi'
 });
-const page = await browser.newPage();
+const page = await chromeBrowser.newPage();
 ...
-await browser.close();
+await chromeBrowser.close();
 ```
 
-This is an exciting step towards a more unified and efficient cross-browser automation experience. We encourage you to explore WebDriver BiDi with Puppeteer and join us in shaping the future of browser automation.
+## Puppeteer features not yet supported over WebDriver BiDi
```

Comment:
Can we add a paragraph here, setting expectations? Specifically, what is the barrier for the "supported only in Chrome" APIs also being available in Firefox.

In that context it might also make sense splitting this section in two, since the headline is saying "yet", but some APIs will likely never make it to BiDi.

Reply:
Removed "supported in Chrome"